src/braket/circuits/gate.py

```diff
@@ -73,6 +73,7 @@ def to_ir(
         *,
         control: Optional[QubitSet] = None,
         control_state: Optional[BasisStateInput] = None,
+        power: Optional[float] = None,
```

Comment:
How about
```suggestion
        power: float = 1,
```
since it's technically true 😛